### DISCUSSION TITLE
Pass request object to Phlex compoent

### DISCUSSION THREAD

**Discussion by ShawnRong** [yippee-fun/phlex#699] [Help] [answered]
Hi team,

I want to get `request.path` variables in a Phlex component. What is the proper way to implement this?

One dummy solution is passing with controller actions method like

``` ruby
  def index
    render Admin::Users::IndexView.new(path: request.path)
  end
```
, but I don't think it is a best practice for this.  

Thanks in advance!

**Comment by joelmoss** [comment, marked as answer]
You can use `helpers.controller.request.path`

**Reply by ShawnRong** [reply]
Thanks for the answer!

**Comment by joeldrapper** [comment]
In the next version of `phlex-rails`, which will be released in the next couple of weeks, you’ll be able to include `Phlex::Rails::Helpers::Request` and then use `request`.

**Reply by ShawnRong** [reply]
Great! Looking forward to the next version of `phlex-rails`!
